apps/frontend/src/pages/Homepage.tsx

```diff
@@ -1,5 +1,246 @@
+        price: 55,
+        dateCreated: "2024-01-05T11:30:00Z",
+        imageUrl: "https://picsum.photos/200/304",
+      },
+      {
+        listingID: "F78K12379G28",
+        sellerID: "F78K12379G28",
+        sellerName: "Charles Boyle",
+        title: "Anthropology 101 Textbook",
+        description: "Some dog-eared pages, but very usable.",
+        price: 45,
+        dateCreated: "2023-12-15T10:30:00Z",
+        imageUrl: "https://picsum.photos/200/305",
+      },
+      {
+        listingID: "G89L23489H29",
+        sellerID: "G89L23489H29",
+        sellerName: "Hitchcock",
+        title: "World History Textbook",
+        description: "Good condition, minimal notes inside.",
+        price: 50,
+        dateCreated: "2023-11-20T09:30:00Z",
+        imageUrl: "https://picsum.photos/200/306",
+      },
+      {
+        listingID: "H90M34599I30",
+        sellerID: "H90M34599I30",
+        sellerName: "Scully",
+        title: "Basic Algebra Textbook",
+        description: "Cover slightly damaged, content intact.",
+        price: 40,
+        dateCreated: "2023-10-18T08:30:00Z",
+        imageUrl: "https://picsum.photos/200/307",
+      },
+    ]);
+  };
+
+  useEffect(() => {
+    fetchListings();
+  }, []);
+
+  return (
+    <Box sx={classes.HomePageBox}>
+      <Box
+        sx={{
+          position: "sticky",
+          top: 0,
+          zIndex: 1000,
+          backgroundColor: "white",
+          padding: "10px 0",
+        }}
+      >
+        <SearchBar onSearch={handleSearch} sortBy={sortBy} />
+      </Box>
+      {searchPerformed ? (
+        <Box
+          sx={{
+            display: "flex",
+            flexDirection: "row",
+            paddingTop: "10px",
+            paddingBottom: "10px",
+          }}
+        >
+          <Typography
+            variant="h5"
+            component="h3"
+            gutterBottom
+            sx={{ margin: "10px" }}
+          >
+            Results ({listingObjects.length})
+          </Typography>
+          <FormControl sx={{ minWidth: "20vw" }}>
+            <Select onChange={handleSortBy} displayEmpty>
+              <FormHelperText>Sort By</FormHelperText>
+              <MenuItem value={"RELEVANCE"}>Relevance</MenuItem>
+              <MenuItem value={"PRICE_ASC"}>Price Ascending</MenuItem>
+              <MenuItem value={"PRICE_DESC"}>Price Descending</MenuItem>
+              <MenuItem value={"LISTED_TIME_ASC"}>
+                Listed Time Ascending
+              </MenuItem>
+              <MenuItem value={"LISTED_TIME_DESC"}>
+                Listed Time Descending
+              </MenuItem>
+              <MenuItem value={"DISTANCE_ASC"}>Distance Ascending</MenuItem>
+              <MenuItem value={"DISTANCE_DESC"}>Distance Descending</MenuItem>
+            </Select>
+          </FormControl>
+        </Box>
+      ) : (
+        <>
+          <Box
+            sx={{
+              borderBottom: "1px solid #808080",
+              fontSize: "1.3em",
```

Comment:
font sizes should be declared either in em or px, but not both. em is probably the better option but px seems to be what is used through most of the front end. Some devices will have different default font sizes, and em makes the font size relative to the parent element, so it can look weird on some screens if not carefully managed

Reply:
changed font-size to px